dotCMS/src/main/webapp/html/portlet/ext/clusterconfig/cluster_config_right_panel.jsp

```diff
@@ -71,9 +76,51 @@
 		</tr>
 		<tr>
 			<td></td>
-			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_es_port") %></td>
-			<td align="right" nowrap>{es.port}</td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_active_shards") %></td>
+			<td align="right">{es.activeShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_relocating_shards") %></td>
+			<td align="right">{es.relocatingShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_initializing_shards") %></td>
+			<td align="right">{es.initializingShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_unassigned_shards") %></td>
+			<td align="right">{es.unassignedShards}</td>
 		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_delayed_unassigned_shards") %></td>
+			<td align="right">{es.delayedUnasignedShards}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_number_of_pending_tasks") %></td>
+			<td align="right">{es.numberOfPendingTasks}</td>
+		</tr>
+		<tr>
+			<td></td>
+			<td><%= LanguageUtil.get(pageContext, "configuration_cluster_number_of_in_flight_fetch") %></td>
+			<td align="right">{es.numberOfInFlightFetch}</td>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid having style information in JSP files.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)